1.8/s2i/bin/assemble

```diff
@@ -9,9 +9,17 @@ if [ -d ./nginx-cfg ]; then
   echo "---> Copying nginx configuration files..."
   if [ "$(ls -A ./nginx-cfg/*.conf)" ]; then
     cp -v ./nginx-cfg/*.conf "${NGINX_CONFIGURATION_PATH}"
+    chmod -Rf 777 ${NGINX_CONFIGURATION_PATH}
```

Comment:
> - Change file owner to default (risky as container can use different user)
> - Make files writable by others by setting 666 👿 mask on file

I am not sure, but I think GID remains the same. So some other images `g+rw` is used to allow running with arbitrary user.